remme/clients/account.py

```diff
@@ -64,13 +70,42 @@ def get_account(self, address):
         account.ParseFromString(self.get_value(address))
         return account
 
+    def get_pubkey_account(self, address):
+        pubkey_acc = PubKeyAccount()
+        pubkey_acc.ParseFromString(self.get_value(address))
+        return pubkey_acc
+
     def get_pub_keys(self, address):
+        future_to_address = {}
+        pub_keys = []
         try:
             account = self.get_account(address)
-            return list(account.pub_keys)
         except KeyNotFound:
             return []
 
+        if not account.pub_key_nonce:
```

Comment:
Same for nonce as in [this comment](https://github.com/Remmeauth/remme-protobuf/pull/2/files#r200629439).